packages/cli/src/constructs/check-codegen.ts

```
@@ -0,0 +1,182 @@
+    this.checkGroupCodegen = new CheckGroupCodegen(program)
+    this.heartbeatCheckCodegen = new HeartbeatCheckCodegen(program)
+    this.multiStepCheckCodegen = new MultiStepCheckCodegen(program)
+    this.tcpCheckCodegen = new TcpCheckCodegen(program)
+  }
+
+  gencode (logicalId: string, resource: CheckResource, context: Context): void {
+    const { checkType } = resource
+
+    switch (checkType) {
+      case 'BROWSER':
+        return this.browserCheckCodegen.gencode(logicalId, resource as BrowserCheckResource, context)
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)